.github/workflows/build-cli.yml

```diff
@@ -14,9 +14,14 @@ jobs:
     name: build-cli
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Harden Runner
+        uses: step-security/harden-runner@1b05615854632b887b69ae1be8cbefe72d3ae423 # v2.6.0
```

Comment:
so it is pinning to a hash, sure more safe. let's hope renovate doesn't batter this in favor of version tag.

Reply:
This is what `"helpers:pinGitHubActionDigests"` Renovate preset handles. It uses the same notation (adding the version in a comment, and pinning to the commit sha).

Reply:
lovely so this is a non issue